src/raw/requests.rs

```diff
@@ -190,23 +192,44 @@ impl KvRequest for kvrpcpb::RawBatchPutRequest {
 }
 
 impl Shardable for kvrpcpb::RawBatchPutRequest {
-    type Shard = Vec<kvrpcpb::KvPair>;
+    type Shard = Vec<(kvrpcpb::KvPair, u64)>;
 
     fn shards(
         &self,
         pd_client: &Arc<impl PdClient>,
     ) -> BoxStream<'static, Result<(Self::Shard, RegionStore)>> {
+        // Maintain a map of the pair and its associated ttl
+        let kvs = self.pairs.clone();
+        let kv_pair = kvs.into_iter().map(Into::<KvPair>::into);
+        let kv_ttl = kv_pair.zip(self.ttls.clone()).collect::<HashMap<_, _>>();
```

Comment:
`store_stream_for_keys` is able to handle data of keys along with values and ttls. Mapping by Hash map is not necessary, and it's not efficient as well.

Please refer to [`PrewriteRequest::shards`](https://github.com/tikv/client-rust/blob/54fd72001bca74288d02134809424aeea7d3f19d/src/transaction/requests.rs#L290).

Reply:
Thanks for the review. Didn't realize `store_stream_for_keys` has this capability. Removed the HashMap to collect the ttls for the kv_pair.

Reply:
Hi @pingyu, I have made the changes. Please let me know if there's anything else I need to consider for this PR.